src/azul/service/manifest_service.py

```diff
@@ -948,10 +948,16 @@ def write_to(self, output: IO[str]) -> Optional[str]:
                                 for field_name in self.manifest_config[source]]
         writer = csv.DictWriter(output, ordered_column_names, dialect='excel-tab')
         writer.writeheader()
+        project_short_names = set()
         for hit in self._create_request().scan():
             doc = self._hit_to_doc(hit)
             assert isinstance(doc, dict)
             file_ = one(doc['contents']['files'])
+            if len(project_short_names) < 2:
+                project_name = one(doc['contents']['projects'])
+                project_name = project_name['project_short_name']
+                if len(project_name) == 1:
```

Comment:
If a file has two project short names, the return value should be `None` since there isn't a single consistent short name, there are at least two. I don't think this does that.